doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+You can use Operator to install Dashbuilder Standalone independently from other services.
+
+.Procedure
+
+. On the *Installation* page, in the *Application name* field, enter `standalone-dashbuilder`.
+. In the *Environment* field, enter `rhpam-standalone-dashbuilder`.
+. Click *Next*.
+. On the *Security* page, click *Next*.
+. On the *Components* page, from the *Components* list, select *Dashbuilder*.
+. To add a {KIE_SERVER} dataset, perform the following tasks:
```

Comment:
Not sure if "tasks" is the best word. Emily may be able to educate us on this. 

I lean more towards "complete the following procedure"

Reply:
I searched for alternative wording and found the most used is "the following tasks" and "do" rather than "perform".

Reply:
"do the following tasks"?